### DISCUSSION TITLE
Generated Azure functions from HTTP triggers from OpenAPI V2/V3 Specification doesn't work

### DISCUSSION THREAD

**Discussion by mmonedero1** [microsoft/vscode-discussions#285] [Extension Development QnA] [answered]
Good morning,

I'm using Azure functions from HTTP triggers from OpenAPI V2/V3 Specification in order to generated a function from a swagger.json(https://petstore.swagger.io/v2/swagger.json).
When i do that, the vsc app response is:
_9:09:32: Using the plugin could overwrite your custom changes to the functions.
If autorest fails, you can run the script on your command-line, or try resetting autorest (autorest --reset) and try again.
9:09:32: Running command: "autorest --input-file:"c:\Users\USER\OneDrive\Documentos\Pruebas Python\FUNCTIONS\pets.json" --version:3.0.6320 --azure-functions-python --no-namespace-folders:True --no-async --generate-metadata:false --output-folder:"c:\Users\USER\OneDrive\Documentos\Pruebas Python\FUNCTIONS\function-dato\api""...
AutoRest code generation utility [cli version: 3.6.2; node: v18.12.1]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
info    | AutoRest core version selected from configuration: 3.0.6320.
   Loading AutoRest core      'C:\Users\USER\.autorest\@autorestcore@3.0.6320\nodemodules\@autorest\core\dist' (3.0.6320)
   Loading AutoRest extension '@autorest/azure-functions-python' (latest->0.1.0-preview)
   Loading AutoRest extension '@autorest/modelerfour' (4.15.414->4.15.414)
TypeError: extension.updatePythonPath is not a function
Process() cancelled due to failure 
9:09:38: Error: Failed to run "autorest" command. Check output window for more details._
¿What is the problem?

I have already try "autorest --reset".

**Comment by isidorn** [comment, marked as answer]
I suggest to open this issue here https://github.com/microsoft/vscode-azuretools
